utils/importJSONFile.js

```diff
@@ -11,16 +12,29 @@ import path from "path";
 async function importJSONFile(filePath, url) {
   if (!filePath) throw new Error("No file path provided");
 
+  const platform = os.platform();
   let finalPath = "";
 
   if (typeof url === "string" && url.length > 0)
-    finalPath = url.split("/").slice(0, -1).join("/").replace("file://", "");
+    finalPath =
+      platform === "win32"
+        ? url
+            .split("/")
+            .slice(0, -1)
+            .join("/")
+            .replace(/^file:\/\/\/?/, "")
+        : url.split("/").slice(0, -1).join("/").replace("file://", "");
```

Comment:
Also here we could move this logic to an util (or can be another exported function in this same file), and add it a jsdoc describing what it does so we can reuse it in case it is needed